doc/eclipse.html

```diff
@@ -87,12 +96,27 @@ <h2 align="center">TestNG Eclipse plug-in</h2>
 <img border="0" src="http://testng.org/pictures/launch.png" />
 </p>
 <p>You should change the name of this configuration and pick a project, which 
-can be selected by clicking on the <tt>Browse...</tt> button at the top of the window.</p><p>Then you choose to launch your TestNG tests in the following ways:</p><h4><a name="eclipse-launch-class">4.2.1 - From a class file</a></h4><p>Make sure the box near <tt>Class</tt> is checked and then pick a class from your 
+can be selected by clicking on the <tt>Browse...</tt> button at the top of the window.</p>
+<p>
+Runtime options:
+<ul>
+<li><b>Log Level</b>: specify the value (0-10) for different verbose log levels</li>
+<li><b>Verbose</b>: enable the runtime TestNG verbose log</li>
+<li><b>Debug</b>: enable more runtime TestNG debug info</li>
+<li><b>Serialization Protocol</b>: the serialization protocol used for communicating between TestNG Eclipse Plugin and TestNG runtime.
+By default it's "Object Serialization", the other option is "String Serializatio" but is deprecated.</li>
```

Comment:
"The default is "Object Serialization", the other option ("String Serialization") is deprecated.